client/stylesheets/_toCat.scss

```diff
@@ -4850,13 +4862,23 @@ On click the images will be displayed at normal size to complete the effect
 	left:12px;
 }
 
+.optionsItems li a:hover {
+	background-color: #d3d3d3 !important;
```

Comment:
Color literals like `#d3d3d3` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented with spaces, not tabs